src/kili/use_cases/tag/__init__.py

```diff
@@ -95,3 +97,18 @@ def get_tag_ids_from_labels(self, labels: Sequence[str]) -> List[TagId]:
             tag_ids.append(TagId(tag_label_to_id[label][0]))
 
         return tag_ids
+
+    def update_tag(
+        self, tag_name: Optional[str], tag_id: Optional[str], new_tag_name: str
+    ) -> UpdateTagReturnData:
+        """Update tag.
+
+        This operation is organization-wide.
+        The tag will be updated for all projects of the organization.
+        """
+        if tag_id is None:
+            if tag_name is None:
+                raise ValueError("Either `tag_name` or `tag_id` must be provided.")
+            tag_id = self.get_tag_ids_from_labels(labels=(tag_name,))[0]
```

Comment:
see: here use cases are calling each others

Reply:
yep this one is a bad copy paste, I fixed it